submissions/OwoeyeGbenga/database/database.txt

```diff
@@ -0,0 +1,4 @@
+Database Task.
+Owoeye Gbenga
+Dufuna Codecamp
+https://docs.google.com/document/d/1wBI3s-6C9qKkA780ajQtCREGO0io3POCXS3SwvPOOqI/edit#
```

Comment:
Please grant read permission on this doc

Reply:
I have granted permission. You can reopen now